src/events/messageReactionAdd.ts

```diff
@@ -10,7 +10,9 @@ export default async (client: Discord.Client, reaction: Discord.MessageReaction,
 
```

Comment:
The use of `reaction.emoji` appears to be growing to be significant, not only here but later in the function. Maybe something like this and use that instead?
```suggestion
const { emoji } = reaction

```